<launch>
  

  <!-- BEGIN ROBOT 1-->
  <!-- <group ns="robot1"> -->
    <!-- <param name="tf_prefix" value="robot1_tf" /> -->
  <include file="$(find multi_robot)/launch/one_robot.launch" >
    <arg name="ns_" value="robot1"/>
    <arg name="ns_tf" value="robot1_tf"/>
    <arg name="x_pos" value="-0.7"/>
    <arg name="y_pos" value="0.0"/>
    <arg name="z_pos" value="0.0"/>
    <arg name="robot_name"  value="Robot1" />
  </include>
  <!-- </group> -->

  <!-- BEGIN ROBOT 2-->
  <!-- <group ns="robot2">
    <param name="tf_prefix" value="robot2_tf" /> -->
    <include file="$(find multi_robot)/launch/one_robot.launch" >
      <arg name="ns_" value="robot2"/>
      <arg name="ns_tf" value="robot2_tf"/>
      <arg name="x_pos" value="-1.0"/>
      <arg name="y_pos" value="1.0"/>
      <arg name="z_pos" value="0.0"/>
      <arg name="robot_name"  value="Robot2" />
    </include>
  <!-- </group> -->

</launch>
